modules/data-table/test/spec.coffee

```diff
@@ -405,7 +400,7 @@ describe 'data-table', ->
           dt.expandedRows().should.eql([])
 
           clickHandlers = container.select('.hx-sticky-table-header-left').select('tbody').selectAll('.hx-data-table-collapsible-toggle')
-            .nodes.map(fakeNodeEvent)
+            .nodes.map(testHelpers.fakeNodeEvent)
```

Comment:
This is what the comment is referring to in `test-helpers.js`:
`// we're using fakeNodeEvent for mapping arrays, so the eventName might be a an index`

The second argument in `map` is the index - we should probably change this to remove the need for checking isNaN:
```
.nodes.map((node) -> testHelpers.fakeNodeEvent(node)) 
```